bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder",
+        output_format="regex",
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'\[(?P<severity>\d)\]\s*'
+                     r'\((?P<origin>.+)\) (?P<message>.+)',
+        severity_map={"1": RESULT_SEVERITY.INFO,
+                      "2": RESULT_SEVERITY.INFO,
+                      "3": RESULT_SEVERITY.NORMAL,
+                      "4": RESULT_SEVERITY.NORMAL,
+                      "5": RESULT_SEVERITY.MAJOR})
```

Comment:
hm maybe you could add a prerequisite check and fail message, such as CSSAutoPrefixBear has for example? :) wouldn't hurt. you could mention in that message that you need to run flawfinder with python2